src/pages/Confirm/index.jsx

```diff
@@ -87,9 +112,7 @@ const Confirm = () => {
         <Form onSubmit={e => submitConfirmation(e)}>
           {({ handleFormChange, value }) => (
             <>
-              <FormInput placeholder="Nombre" name="name" value={value} onChange={handleFormChange} />
               <FormInput placeholder="RUT" name="RUT" value={value} onChange={handleFormChange} />
-              <FormInput placeholder="Celular" name="phone" value={value} onChange={handleFormChange} />
```

Comment:
Why here is only using the RUT? @Israel-Laguan is a change of the customer?

Reply:
Here's happening same. @Israel-Laguan and me chatted about this and did it this